lib/Translator.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+// Copyright 2022. Plesk International GmbH. All rights reserved.
+
+namespace WHMCS\Module\Server\Plesk360Monitoring;
+
+class Translator
```

Comment:
whole class can be switched to Singletone pattern with single init method

Reply:
Done